### ISSUE TITLE
How to improve diversity of generated images?

### ISSUE THREAD

**Issue by znavidi** [FoundationVision/Infinity#112]
Thanks for the excellent tool and open-source resources.

I fine-tuned the Infinity 2B model on my custom dataset, achieving reasonable image quality with improving loss and accuracy. However, during inference, varying the seed results in only minor differences between generated images. Since diversity is important for my use case, are there specific parameters I can tweak to enhance output variability for a given input condition? or any other suggestions in general would be appreciated.

**Comment by JeyesHan** [comment]
Hi, znavidi. Setting a lower cfg value will improve diversity. Besides, avoiding overfitting your fine-tuning dataset will also be beneficial to generation diversity.

**Comment by znavidi** [comment]
Thanks a lot for your guidance. will close the issue.
